ironfish/src/migrations/migrator.ts

```diff
@@ -68,7 +68,7 @@ export class Migrator {
   async revert(options?: { dryRun?: boolean }): Promise<void> {
     const dryRun = options?.dryRun ?? false
 
-    const migrations = Array.from(this.migrations).reverse()
+    const migrations = this.migrations.slice().reverse()
```

Comment:
Is this a style-related change, or do these have functional differences?

Reply:
no functional difference, I just made it the same since I was fixing other bugs.